EIPS/eip-131.md

```diff
@@ -0,0 +1,87 @@
+---
+eip: 131
+title: Blake2b Ethereum Implementation
+author:  Tjaden Hess (@tjade273) , Jay Graber , James Hancock (@madeoftin)
```

Comment:
Nitpicking: please remove the extra spaces.

Reply:
I think I have them all removed. Not sure what the bot is looking for so let me know.